.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: node_js
-sudo: false
+services:
```

Comment:
Ok. This might have changed since the last time I looked at it? Not sure. It seems PKI is used for this and the private key is associated to the project in Travis and inaccessible to anyone. 👍 And forks of this project all get their own key pair. Even better.

So this is safe and definitely way preferable to using the Travis UI to set this.